book.bib

```diff
@@ -548,6 +548,21 @@ @Manual{R-TreeSummarizedExperiment
   note =	 {R package version 1.6.2},
 }
 
+@Manual{R-MultiAssayExperiment,
+  title =	 {MultiAssayExperiment: Software For The Integration Of Multi-Omics
```

Comment:
Put MultiAssayExperiment in curly brackets "{MultiAssayExperiment}" -> this will preserve capitalization in different output layouts. That is essential because this is a name.